example/javascriptlib/basic/5-client-server-hello/client/src/client.ts

```diff
@@ -0,0 +1,51 @@
+import * as http from 'http';
```

Comment:
Let's make this a web client rather than a node.js client, and have it served from the server the same was the todomvc example